docs/content/v2.0/admin/yb-admin.md

```diff
@@ -534,18 +534,23 @@ yb-admin -master_addresses <master-addresses> modify_placement_info <placement_i
 ```
 
 - *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
-- *placement_info*: Comma-delimited list of placements for *cloud*.*region*.*zone*. Default value is `cloud1.datacenter1.rack1`.
-- *replication_factor*: The number of replicas for each tablet.
+- *placement_info*: Comma-delimited list of placements for *cloud*.*region*.*zone*. Default value is `cloud1.datacenter1.rack1`. Optionally, a minimum replica count can be specified after a colon, default being 1. If a placement is specified multiple times, the total count from all mentions are taken.
```

Comment:
Rephrase the last line to:  It is not recommended to repeat placement multiple times but instead specify the total count after the colon. However, in the event that the user specifies a placement multiple times, the total count from all mentions is taken.